app/Http/Controllers/Api/ClientApiController.php

```diff
@@ -333,7 +333,7 @@ public function revokeToken($id, $value, $hint)
                     if ($token->getClientId() !== $client->getClientId()) {
                         throw new ValidationException(sprintf('access token %s does not belongs to client id !', $value, $id));
                     }
-                    $this->token_service->revokeAccessToken($value, true);
+                    //$this->token_service->revokeAccessToken($value, true);
```

Comment:
this breaks functionality please rollback

Reply:
done, sorry